packages/example-codehub/src/codehub-application.ts

```diff
@@ -25,13 +28,13 @@ export class CodeHubApplication extends Application {
     app.controller(UserController);
     app.controller(HealthController);
 
-    app.bind('userId').to(42);
+    server.bind('userId').to(42);
```

Comment:
I think `userId` is application specific, not server/transport specific, therefore it should be bound in `app` context.